util/topgen/templates/toplevel_racl_pkg.sv.tpl

```diff
@@ -0,0 +1,82 @@
+// Copyright lowRISC contributors (OpenTitan project).
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+${gencmd}
+
+package top_racl_pkg;
+  // Number of RACL policies used
+  parameter int unsigned NrRaclPolicies = ${racl_config['nr_policies']};
+
+  // Number of RACL bits transferred
+  parameter int unsigned NrRaclBits = 4;
+
+  // Number of CTN UID bits transferred
+  parameter int unsigned NrCtnUidBits = 8;
+
+  // RACL role type binary encoded
+  typedef logic [NrRaclBits-1:0] racl_role_t;
+
+  // CTN UID assigned the bus originator
+  typedef logic [NrCtnUidBits-1:0] ctn_uid_t;
+
+  // RACL permission and one-hot encoded role vector
+  typedef logic [(2**NrRaclBits)-1:0] racl_role_vec_t;
+
+  // RACL policy vector containing a read and write permission
```

Comment:
I'm not sure that "policy vector" is the right name here: there are two vectors...

Reply:
Let's drop that to policy.